documentation/ember-cli-build.js

```diff
@@ -8,6 +8,14 @@ const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
+    sassOptions: {
+      extension: 'scss',
+      includePaths: [
+        '../node_modules/@hashicorp/design-system-components/app/styles',
+        '../node_modules/@hashicorp/consul-ui-toolkit/dist/styles',
+        '../node_modules/@hashicorp/design-system-tokens/dist/products/css',
```

Comment:
Not sure if the order matters, but just in case...
```suggestion
        '../node_modules/@hashicorp/design-system-tokens/dist/products/css',
        '../node_modules/@hashicorp/design-system-components/app/styles',
        '../node_modules/@hashicorp/consul-ui-toolkit/dist/styles',
```